packages/storage/amplify_storage_s3_dart/lib/src/model/s3_item.dart

```diff
@@ -44,7 +43,10 @@ class S3Item extends StorageItem {
 
     // In S3 plugin, key is required property presenting an object
     if (key == null) {
-      throw S3Exception.getS3ObjectMissingKeyException(object);
+      throw const UnknownException(
```

Comment:
For comment above, did you mean: 

    // In S3 plugin, key is required property presenting an object

    // In S3 plugin, key is required property present in object